index.html

```diff
@@ -19,7 +19,7 @@
 	<script> 
 	var frankly_menu = [
 	'<ul>',
-		'<li><a href="index.html">Getting Started</a></li>',
+		'<li><a href="http://franklyinc.github.io">Getting Started</a></li>',
```

Comment:
Yeah, my goal is to eliminate franklyinc.github.io becoming franklyinc.github.io/index.html when somebody goes back home. This is purely superficial. I researched how to do this and everything I read implied that the way to make that sort of superficial change happen is by modifying your .htaccess config but I don't have such a file with a github.io page. I'm open to suggestions here - not married to any solution, just hoping for the end result of prettier URLs.